eox_core/api/task_dispatcher/v1/views.py

```diff
@@ -0,0 +1,117 @@
+"""
+View for the tasks dispatcher API
+"""
+import logging
+from importlib import import_module
+
+import celery
+from django.conf import settings
+from django.contrib.sites.shortcuts import get_current_site
+from rest_framework.permissions import IsAdminUser
+from rest_framework.response import Response
+from rest_framework.status import HTTP_404_NOT_FOUND
+from rest_framework.views import APIView
+
+from eox_core.edxapp_wrapper.bearer_authentication import BearerAuthentication
+
+LOG = logging.getLogger(__name__)
+
+
+class TaskAPI(APIView):
+    """
+    Celery task dispatcher API
+    """
+
+    authentication_classes = [BearerAuthentication]
+    permission_classes = [IsAdminUser]
```

Comment:
I wonder if we could have a use case where we need this for a user with a specific permission for a site.

Reply:
For the moment I just assumed that the current tasks that could be possibly be run would at least require that level of authorization, and other APIs use those permissions, the eox-core permissions class was obviously discarded because clients have access to those credentials. Maybe in the future we can add a new permission class if needed.